grails-app/assets/javascripts/organisation.js

```diff
@@ -9,11 +11,96 @@
 OrganisationViewModel = function (props, options) {
     var self = $.extend(this, new Documents(options));
 
+    var coreReportService = new CoreReportService(options)
+    _.extend(this, coreReportService);
+
     var defaults = {
         validationContainerSelector: '.validationEngineContainer',
     };
 
-    var config = _.extend({}, defaults, options);
+    var config = props.config || {};
+    self.config = ko.observable(vkbeautify.json(props.config));
+
+    self.organisationReportCategories = ko.computed(function() {
+        return _.map(config.organisationReports || [], function(report) {
+            return report.category;
+        });
+    });
+
+    var coreServicesReportCategory = 'Core Services Reporting';
+    var getOrganisationReportConfig = function() {
+        if (!config.organisationReports || config.organisationReports.length == 0) {
+            config.organisationReports = [{type:'Administrative', category:coreServicesReportCategory}];
+        }
+        return config.organisationReports[0];
+    };
+
+    var organisationReportConfig = getOrganisationReportConfig();
+
+    self.coreServicesOptions = [
+        {label:'Monthly (First period ends 31 July 2018)', firstReportingPeriodEnd:'2018-07-31T14:00:00Z', reportingPeriodInMonths:1, reportConfigLabel:'Monthly'},
```

Comment:
We need to think about these options - the latest information is they will all be quarterly starting September.
For now, narrow them down to one choice, First period ends 30 September 2023.